src/XRFeitoriaBpy/core/factory.py

```diff
@@ -998,28 +998,17 @@ def import_glb(glb_file: str) -> None:
         except Exception as e:
             raise Exception(f'Failed to import glb: {glb_file}\n{e}')
 
-    def import_mo_json(
-        cls,
-        mo_json_file: Path,
-        actor_name: str,
-    ) -> None:
+    def import_mo_json(mo_json_file: Path, actor_name: str) -> None:
         """Import an animation from json, and apply the animation to the given actor.
 
         Args:
             mo_json_file (Path): json file path.
             actor_name (str): Name of the actor.
         """
         motion_data = json.load(Path(mo_json_file).open('r'))
```

Comment:
Opening a file without close.

Reply:
Fixed in [`b54c643` (#12)](https://github.com/openxrlab/xrfeitoria/pull/12/commits/b54c643570638bb0ed3d5b31cdbfaad1ba086b45#diff-0cae5b014ff6e456cc55ea817c51670678870333f7a3cb0b17a46fa974562ebf)